octobot.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    service: octobot
+  name: octobot
+spec:
+  ports:
+    - port: 5001
+  selector:
+    service: octobot
+  type: LoadBalancer
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    service: octobot
+  name: octobot
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      service: octobot
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        service: octobot
+    spec:
+      containers:
+        - image: drakkarsoftware/octobot:stable
+          name: octobot
+          ports:
+            - containerPort: 5001
+              name: octobot
+          volumeMounts:
+            - mountPath: /octobot/logs
+              name: octobot-logs
+            - mountPath: /octobot/backtesting
+              name: octobot-backtesting
+            - mountPath: /octobot/tentacles
+              name: octobot-tentacles
+            - mountPath: /octobot/user
+              name: octobot-user
+      restartPolicy: Always
+      volumes:
+        - name: octobot-logs
+          persistentVolumeClaim:
+            claimName: octobot-logs
+        - name: octobot-backtesting
+          persistentVolumeClaim:
+            claimName: octobot-backtesting
+        - name: octobot-tentacles
+          persistentVolumeClaim:
+            claimName: octobot-tentacles
+        - name: octobot-user
+          persistentVolumeClaim:
+            claimName: octobot-user
+status: {}
+
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  labels:
+    service: octobot
+  name: octobot-logs
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 100Mi
+status: {}
+
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  labels:
+    service: octobot
+  name: octobot-backtesting
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 100Mi
+status: {}
+
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
```

Comment:
We may switch to https://kubernetes.io/docs/concepts/storage/ephemeral-volumes/ for `tentacles` volume